ibis/backends/postgres/tests/test_postgis.py

```diff
@@ -149,9 +149,16 @@ def test_geo_disjoint(geotable):
 
 
 def test_geo_equals(geotable):
-    expr = geotable.geo_point.equals(geotable.geo_point)
+    # note: == operation just works for comparison between same datatypes
+    expr = geotable.geo_point == geotable.geo_point
     assert expr.execute().all()
 
+    expr = geotable.geo_point.geo_equals(geotable.geo_point)
+    assert expr.execute().all()
+
+    expr = geotable.geo_point.geo_equals(geotable.geo_linestring)
+    assert not expr.execute().all()
```

Comment:
Do you want all them to be different? Then `any` should be better I think.

Reply:
that makes sense. I will change it now. thanks @datapythonista